integration_test/third_party_apps_data/applications/hadoop/centos_rhel/install

```diff
@@ -0,0 +1,144 @@
+set -e
+
+sudo yum -y install \
+    java-1.8.0-openjdk java-1.8.0-openjdk-devel wget
+
+sudo mkdir -p \
+    /opt/hadoop/logs \
+    /opt/volume/datanode \
+    /opt/volume/namenode
+
+wget https://dlcdn.apache.org/hadoop/common/hadoop-3.3.1/hadoop-3.3.1.tar.gz
```

Comment:
Chances are a bit higher that users will use the default package manager (e.g. yum, apt-get, zyypper) to install a package instead of a pinned version of the application.

Can we adjust it to do something like `yum install hadoop` instead? Similar requests for the other `install` scripts.

Reply:
there doesn't seem to be an available package for hadoop.

Reply:
I agree, I don't see any packaging for Hadoop. This seems like a fairly common thing for Java apps - hopefully we can get away from manual installation when we move on to non-Java apps.

Reply:
Ah NVM for this app then.